tests/posix/pthread_rwlock/src/posix_rwlock.c

```diff
@@ -80,7 +80,8 @@ static void test_rw_lock(void)
 
 	/* Creating N premptive threads in increasing order of priority */
 	for (i = 0; i < N_THR; i++) {
-		pthread_attr_init(&attr[i]);
+		zassert_equal(pthread_attr_init(&attr[i]), 0,
+								"Unable to create pthread object attrib\n");
```

Comment:
this should be aligned with the opening parenthesis of zassert_equal(), not way out over here

Reply:
Updated the patch . Please review .